### PR TITLE
json api error object, error messages, translation for exceptions, translations for texts

### PR DESCRIPTION
**Pull request by gymad**
Documentation added for:
- json api error object
- error messages
- translation for exceptions
- translations for strings

Please do not merge it until if related PR also merged into SuiteCRM repository:
https://github.com/salesagility/SuiteCRM/pull/5528

### CODE REVIEW

**Comment by gymad** [comment]
Requested changes done.

> -These files would be better placed in the developer/API directory, please can you move them there.
These files doesn't related specially to the API. Only the json api error object implementation maybe..

File: content/developer/API/JSON API Error Object.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+Title: JSON API Error Object
+---
+
+== JsonApiErrorObject class
```
Comment (CBeechey) [inline content/developer/API/JSON API Error Object.adoc]: Change to level one heading =

File: content/developer/Error Messages.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Error Messages"
+---
+
+== ErrorMessage class
```
Comment (CBeechey) [inline content/developer/Error Messages.adoc]: Change to level one heading =

File: content/developer/Exceptions and Localization.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+Title: Exceptions and Localization
+---
+
+== LangException class
```
Comment (CBeechey) [inline content/developer/Exceptions and Localization.adoc]: Change to level one heading =

File: content/developer/JSON API Error Object.adoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+Title: JSON API Error Object
+---
+
+== JsonApiErrorObject class
```
Comment (CBeechey) [inline content/developer/JSON API Error Object.adoc]: Change to level one heading =

File: content/developer/Translate strings.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+Title: Translate strings
+---
+
+== LangText class
```
Comment (CBeechey) [inline content/developer/Translate strings.adoc]: Change to level one heading =

**Pull request merged**